tests/tests/test-democracy.ts

```diff
@@ -331,8 +331,8 @@ describeDevMoonbeam("Democracy - forget notePreimage", (context) => {
     encodedHash = blake2AsHex(encodedProposal);
   });
 
-  it("vote", async function () {
-    this.timeout(200000);
+  it("vote foo bar", async function () {
```

Comment:
this name makes it hard to use `npm run test -- --grep` because `vote` matches a lot of tests...

Reply:
(err, I removed `foo bar` -- this comment is about the test being named just "vote")